department-of-veterans-affairs/caseflow
# AOD logic for CO hearings

Resolves #7616 

### Description
This PR adds the AOD logic for central office hearings. 

### To test:
1) On the Assign Hearings page, select Central, and confirm AOD cases appear in order.
